src/System.ServiceModel.Duplex/tests/project.json

```diff
@@ -2,37 +2,22 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.2-beta-24223-03",
     "System.Linq.Expressions": "4.1.1-beta-24223-03",
-    "System.ServiceModel.Duplex": "4.0.2-beta-24229-01",
-    "System.ServiceModel.Http": "4.1.1-beta-24229-01",
-    "System.ServiceModel.NetTcp": "4.1.1-beta-24229-01",
-    "System.ServiceModel.Primitives": "4.1.1-beta-24229-01",
-    "System.ServiceModel.Security": "4.0.2-beta-24229-01",
-    "xunit": "2.1.0",
-    "xunit.netcore.extensions": "1.0.0-prerelease-00312-04",
     "test-runtime": {
       "target": "project",
       "exclude": "compile"
-    }
+    },
+    "Microsoft.xunit.netcore.extensions": "1.0.0-prerelease-00508-01",
+    "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00508-01"
   },
   "frameworks": {
-    "netcoreapp1.0": {
-      "imports": [
-        "dnxcore50",
-        "portable-net45+win8"
-      ]
-    }
+    "netstandard1.3": { }
   },
-  "runtimes": {
-    "centos.7-x64": {},
-    "debian.8-x64": {},
-    "fedora.23-x64": {},
-    "linux-x64": {},
-    "opensuse.13.2-x64": {},
-    "osx.10.10-x64": {},
-    "rhel.7-x64": {},
-    "ubuntu.14.04-x64": {},
-    "ubuntu.16.04-x64": {},
-    "win7-x64": {},
-    "win7-x86": {}
+  "supports": {
+    // "coreFx.Test.netcore50": { }, // System.Net.Security has no run-time assembly compatible with win10-x86 / win10-x86-aot / win10-x64 / win10-x64-aot
```

Comment:
Looks like this isn't valid json. 
eg . "_comment": "comment text goes here...",